src/bindings/python/tests/test_graph/test_ops_scatter.py

```diff
@@ -22,14 +23,25 @@ def test_scatter_update_props():
     assert node.get_output_element_type(0) == Type.i8
 
 
-def test_scatter_update_elements_props():
+@pytest.mark.parametrize(
+    "reduction",
+    [
+        "none",
+        "sum",
+        "prod",
+        "min",
+        "max",
+        "mean",
+    ],
+)
+def test_scatter_update_elements_props(reduction):
     dtype = np.int8
     parameter_r = ov.parameter([2, 4, 5, 7], dtype=dtype, name="data")
     parameter_i = ov.parameter([2, 2, 2, 2], dtype=dtype, name="indices")
     parameter_u = ov.parameter([2, 2, 2, 2], dtype=dtype, name="updates")
     axis = np.array([1], dtype=np.int8)
 
-    node = ov.scatter_elements_update(parameter_r, parameter_i, parameter_u, axis)
+    node = ov.scatter_elements_update(parameter_r, parameter_i, parameter_u, axis, reduction, False)
```

Comment:
Please add additional tests with default values for attributes like:
`node = ov.scatter_elements_update(parameter_r, parameter_i, parameter_u, axis)`